packages/bolt-connection/test/pool/pool.test.js

```diff
@@ -237,6 +237,45 @@ describe('#unit Pool', async () => {
     expect(r0.destroyed).toBeTruthy()
   })
 
+  it('destroys resource when key was purged event when key is acquired again', async () => {
```

Comment:
I don't understand what "event" means in this context.

Reply:
Nothing, It was my dyslexia talking.